src/backend/web/routes/post.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const { getPost } = require('../../utils/storage');
+
+const post = express.Router();
+
+post.get('/:id', async (req, res) => {
+  const redisPost = await getPost(req.params.id);
+
+  /**
+   * If the post retrieved from redis doesn't have the property 'author'
+   * it means the id passed to the endpoint is invalid, and the endpoint sends
+   * a JSON { 'id': '-1' } so the frontend knows that the passed id was invalid.
+   */
+  redisPost.guid = Reflect.has(redisPost, 'author') ? req.params.id : '-1';
```

Comment:
First time I've ever seen `Reflect` used :)

Again, I'm skeptical of `'-1'` here.